### ISSUE TITLE
Update to work with ResStock 3.1/OS-HPXML 1.6.0

### ISSUE THREAD

**Issue by jmaguire1** [NREL/OCHRE#28]
Looks like there's some breaking changes in the most recent release of OS-HPXML. Lighting is the first one we hit, but there's probably several breaking changes. See: https://github.com/NREL/OpenStudio-HPXML/releases/tag/v1.6.0

**Comment by pemami4911** [comment]
Hi,
I'm getting this error trying to load an HPXML file created with ReStock 3.0: 

```
Traceback (most recent call last):
  File "/Users/pemami/Workspace/ochre_gym/test/test_reset_env.py", line 9, in setUp
    self.env = ochre_gym.load(
  File "/Users/pemami/Workspace/ochre_gym/ochre_gym/__init__.py", line 202, in load
    env = OchreEnv(exp_config, dwelling_args, logger)
  File "/Users/pemami/Workspace/ochre_gym/ochre_gym/ochre_env.py", line 51, in __init__
    self.dwelling = Dwelling(experiment_args['env_name'], **dwelling_args)
  File "/opt/miniconda3/envs/ochre_official/lib/python3.9/site-packages/ochre/Dwelling.py", line 71, in __init__
    properties, weather_station = load_hpxml(**house_args)
  File "/opt/miniconda3/envs/ochre_official/lib/python3.9/site-packages/ochre/utils/hpxml.py", line 1558, in load_hpxml
    zones['Indoor'].update(parse_indoor_infiltration(hpxml, construction, equipment_dict))
  File "/opt/miniconda3/envs/ochre_official/lib/python3.9/site-packages/ochre/utils/hpxml.py", line 412, in parse_indoor_infiltration
    inf = indoor_infiltration['AirInfiltrationMeasurement']
KeyError: 'AirInfiltrationMeasurement'
``` 

In my HPXML file, the SystemIdentifier id is `AirInfiltrationMeasurement1` but the code seems to expect `AirInfiltrationMeasurement`.

here's the relevant snipped in my HPXML file:
```
<Enclosure>
      <AirInfiltration>
        <AirInfiltrationMeasurement>
          <SystemIdentifier id='AirInfiltrationMeasurement1'/>
          <HousePressure>50.0</HousePressure>
          <BuildingAirLeakage>
            <UnitofMeasure>ACH</UnitofMeasure>
            <AirLeakage>5.0</AirLeakage>
          </BuildingAirLeakage>
          <InfiltrationVolume>17432.0</InfiltrationVolume>
          <InfiltrationHeight dataSource='software'>8.0</InfiltrationHeight>
        </AirInfiltrationMeasurement>
      </AirInfiltration>
```

Many of the SystemIdentifiers have these digits (1/2/3/etc.) after the name. 
Is this an issue with how the HPXML file was created or with how OCHRE now parses HPXML files?

**Comment by mnblonsky** [comment]
@pemami4911, can you send me the HPXML file? This was one of the errors that I thought we had fixed, but this could be an edge case that we missed.

**Comment by jmaguire1** [comment]
@pemami4911: If you could also send the buildstock.csv file (or post them here) and tell us which run this error occurs for, that would be helpful so we can make sure we get the root cause of the issue.

**Comment by pemami4911** [comment]
Here is the buildstock.csv file (with just the one building in it):
[buildstock.csv](https://github.com/NREL/OCHRE/files/12322849/buildstock.csv)

And the generated HPXML file (remove the .txt, Github doesn't allow .xml uploads):
[home.xml.txt](https://github.com/NREL/OCHRE/files/12322850/home.xml.txt)

**Comment by pemami4911** [comment]
Oh awesome, will switch to using this one. Thanks!

**Comment by mnblonsky** [comment]
@pemami4911 I just pushed a fix. We'll merge that into dev soon. You're welcome to keep using ResStock 3.0 if you prefer.

The issue had to do with how we're parsing the HPXML file. ResStock always included an `extension` in the `AirInfiltration` element, but BEopt does not. When there's only 1 element, we used to use the `SystemIdentifier`, but we didn't if there was 2 or more elements with different names, i.e. `AirInfiltrationMeasurement` and `extension`. That's changed now, and we'll only use the `SystemIdentifier` if there's elements with the same name, e.g. for `Wall` elements. Hopefully this fixes a lot of potential future issues too...
